core/src/main/resources/lib/form/hetero-list.jelly

```diff
@@ -123,10 +127,22 @@ THE SOFTWARE.
       <j:set var="instance" value="${null}" />
       <j:set var="descriptors" value="${h.filterDescriptors(it,attrs.descriptors)}" />
       <j:forEach var="descriptor" items="${descriptors}" varStatus="loop">
-        <div name="${attrs.name}" title="${descriptor.displayName}" tooltip="${descriptor.tooltip}" descriptorId="${descriptor.id}">
+        <j:choose>
+          <j:when test="${titleClassMethod!=null}">
+            <j:set var="classAndMethod" value="${titleClassMethod.split(&quot;#&quot;)}"/>
+            <j:invokeStatic var="symbol" className="${classAndMethod[0]}" method="${classAndMethod[1]}">
+              <j:arg value="${descriptor}" type="hudson.model.Descriptor" />
+            </j:invokeStatic>
+            <j:set var="title" value="${symbol}: ${descriptor.displayName}"/>
+          </j:when>
+          <j:otherwise>
+            <j:set var="title" value="${descriptor.displayName}"/>
+          </j:otherwise>
+        </j:choose>
+        <div name="${attrs.name}" title="${title}" tooltip="${descriptor.tooltip}" descriptorId="${descriptor.id}">
           <j:set var="capture" value="${attrs.capture?:''}" />
           <local:body deleteCaption="${attrs.deleteCaption}">
-            <l:renderOnDemand tag="tr" clazz="config-page" capture="descriptor,it,instance,${capture}">
+            <l:renderOnDemand tag="tr" clazz="config-page" capture="descriptor,it,instance,h,${capture}">
```

Comment:
@jsoref what is h?

Reply:
`h` is implicitly defined by `initPageVariables()`

Reply:
Oh wait, not sure what's going on there. `h` is just a reference to `Functions`.

Reply:
https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/master/pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/generator/DirectiveGenerator/symbol-hetero-list.jelly#L132